pkg/shell/manifest.json.in

```diff
@@ -32,7 +32,8 @@
                 "--disable-internal-agent",
                 "cockpit-bridge",
                 "--privileged"
-            ]
+            ],
+            "poison-on-logout": true
```

Comment:
Lets remove the word "logout". It is a historical accident that a ```"logout"``` message is used to drop privileges. 

I would suggest using "privileged" or some metadata related to the bridge, not related to the action to take.

Reply:
Yes, very good.  Done.